controllers/storagecluster_controller.go

```diff
@@ -45,17 +58,111 @@ type StorageClusterReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (r *StorageClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx)
+
+	instance := &ocsv1.StorageCluster{}
+	err := r.Client.Get(context.TODO(), req.NamespacedName, instance)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			logger.Info("storagecluster instance not found")
```

Comment:
Make sure all instances of `StorageCluster` and `StorageSystem` have proper camelcase.

Reply:
Done